pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -186,6 +191,52 @@ func init() {
 	}
 
 	err = InfraSAs.addServiceAccount(
+		InfraImageTriggerControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: ImageTriggerControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// List Watch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					APIGroups: []string{imageapi.GroupName, imageapi.LegacyGroupName},
+					Resources: sets.NewString("imagestreams"),
+				},
+				// Spec update on triggerable resources
+				{
+					Verbs:     sets.NewString("get", "update"),
+					APIGroups: []string{extensionsGroup},
+					Resources: sets.NewString("daemonsets"),
+				},
+				{
+					Verbs:     sets.NewString("get", "update"),
+					APIGroups: []string{extensionsGroup, appsGroup},
+					Resources: sets.NewString("deployments"),
+				},
+				{
+					Verbs:     sets.NewString("get", "update"),
+					APIGroups: []string{appsGroup},
+					Resources: sets.NewString("statefulsets"),
+				},
+				{
+					Verbs:     sets.NewString("get", "update"),
+					APIGroups: []string{deployapi.GroupName, deployapi.LegacyGroupName},
+					Resources: sets.NewString("deploymentconfigs"),
+				},
+				{
+					Verbs:     sets.NewString("create"),
+					APIGroups: []string{buildapi.GroupName, buildapi.LegacyGroupName},
+					Resources: sets.NewString("buildconfigs/instantiate"),
+				},
+			},
```

Comment:
Deliberately does not have them yet, post rebase I'll add cron jobs as a separate commit / PR